README.md

```diff
@@ -1 +1,8 @@
-# active-flood-viz
\ No newline at end of file
+# active-flood-viz
+
+SVGIS requires Fiona, which in turn requires GDAL, which in turn requires the Visual C++ Build Tools -- which must be
+downloaded from http://landinghub.visualstudio.com/visual-cpp-build-tools
+
+Also, on my machine, it takes about 60 seconds for the SVG to actually be generated.
```

Comment:
These comments (lines 6 and 8) don't belong in a README. README's should contain information about how to set up the project, so lines 3 and 4 are appropriate. Comments about how to run the application may also be included. And descriptions of what the repo does. Configuration parameter description may also be included.